### PR TITLE
Context management

### PR DESCRIPTION
**Pull request by mikbry**
#29 WIP

### CODE REVIEW

**Comment by mikbry** [comment]
#29 DONE

File: src/middlewares/openNLX.js [hunk 1]
```diff
@@ -43,7 +43,11 @@ class OpenNLXMiddleware {
       // reset Conversation / Context
       openNLX.deleteContext(bot.id, v, data.conversationId);
       // delete in db parameters
-      await parameters.deleteValue(data.conversationId);
+      try {
+        await parameters.deleteValue(data.conversationId);
+      } catch (error) {
+        // Silent error
```
Comment (willdurand) [inline src/middlewares/openNLX.js]: shouldn't we log the error in `debug` at least?

Comment (mikbry) [reply src/middlewares/openNLX.js]: In fact it is not an error: the context could be not create as it was not used. That's why I don't log the error

Comment (willdurand) [reply src/middlewares/openNLX.js]: I see, maybe improve the comment to explain why it is ok to silent this error.

**Pull request merged**